### DISCUSSION TITLE
doesn't work underline syntax markdown

### DISCUSSION THREAD

**Discussion by l1mindr** [remarkjs/remark#1011] [Q&A] [answered]
When I type \~~underline~~ the text inside the p tag is displayed
Why??

**Comment by wooorm** [comment]
That is not markdown.

**Reply by l1mindr** [reply]
so what?

**Reply by wooorm** [reply]
We parse markdown. That is not markdown.

**Reply by l1mindr** [reply]
So this is not defined?

**Reply by wooorm** [reply]
No.

**Comment by ChristianMurphy** [comment, marked as answer]
Because it isn't CommonMark markdown, which remark supports by default, this is what's supported in CommonMark: https://commonmark.org/help/

And in GitHub Flavored Markdown: `~~` is a strike through, not underline https://github.github.com/gfm/#strikethrough-extension-

No flavor of markdown I'm aware of uses `~~` as an underline (<https://babelmark.github.io/?text=~~test~~>).
Though you could make a plugin to add this if you really want: https://unifiedjs.com/learn/guide/
